### PR TITLE
release: v1.7.0

### PR DESCRIPTION
**Pull request by rhamzeh**
## Review Checklist
- [x] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [x] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

If you haven't done so yet, we would appreciate it if you could star the [OpenFGA repository](https://github.com/openfga/openfga). :)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,14 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
 
+## [1.7.0] - 2024-10-29
+
 ### Added
 * Documenting OpenFGA release process [#1923](https://github.com/openfga/openfga/pull/1923)
 * Cache Controller to extend Sub-problems and Iterators lifetime in cache [#2006](https://github.com/openfga/openfga/pull/2006)
-* Add access control experimental feature [#1913](https://github.com/openfga/openfga/pull/1913)
+* Add an experimental access control feature [#1913](https://github.com/openfga/openfga/pull/1913)
+  Learn more about this feature and how to enable it [here](https://openfga.dev/docs/getting-started/setup-openfga/access-control)
+  If you do try it out, please provide feedback in the [GitHub Discussion](https://github.com/orgs/openfga/discussions/378)
```
Comment (rhamzeh) [inline CHANGELOG.md:18]: Haven't created the discussion yet, will open it once we're set release

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,10 +8,14 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
...
 ## [Unreleased]
 
+## [1.7.0] - 2024-10-29
+
 ### Added
 * Documenting OpenFGA release process [#1923](https://github.com/openfga/openfga/pull/1923)
 * Cache Controller to extend Sub-problems and Iterators lifetime in cache [#2006](https://github.com/openfga/openfga/pull/2006)
-* Add access control experimental feature [#1913](https://github.com/openfga/openfga/pull/1913)
+* Add an experimental access control feature [#1913](https://github.com/openfga/openfga/pull/1913)
+  Learn more about this feature and how to enable it [here](https://openfga.dev/docs/getting-started/setup-openfga/access-control)
```
Comment (rhamzeh) [inline CHANGELOG.md:17]: This Doc is not yet public, we'll merge once released. The preview is [here](https://openfga.dev/pr-preview/pr-862/docs/getting-started/setup-openfga/access-control)

**Review by jpadilla — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (jpadilla) [reply CHANGELOG.md:18]: @rhamzeh will that link continue being correct when you create the discussion? right now it points to takes you to https://github.com/openfga/community/pull/378

Comment (rhamzeh) [reply CHANGELOG.md]: Pointing to the full discussions for now and we can update after release

**Review by jpadilla — DISMISSED** [review]

**Review by jpadilla — APPROVED** [review]

**Review by Vic-Dev — APPROVED** [review]

**Pull request merged**